sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -52,9 +52,15 @@ case class CatalogStorageFormat(
     properties: Map[String, String]) {
 
   override def toString: String = {
+    val maskedProperties = properties.map {
+      case (password, _) if password.toLowerCase == "password" => (password, "###")
```

Comment:
Yeah, you are right. 

At the beginning, I also thought maybe we just need to hide the password value instead of hiding the whole URL. Later, when I read another PR: https://github.com/apache/spark/pull/10452, I thought maybe we can hide the whole `url`. After rethinking it, users might need to see the other field values. Will follow your suggestions to hide password values only. Thanks!